ortho_seq_code/plotclass.py

```diff
@@ -0,0 +1,326 @@
+import os
+import numpy as np
+from ortho_seq_code.constants_orthoseqs import *
+import matplotlib.pyplot as plt
+import click
+
+
+def create_dir_if_not_exists(out_dir):
```

Comment:
I left feedback on the other copy of this function, but it looks like this function is duplicated so this should just use the existing version. It might be good to pull it out into a shared, common, utilities, tools, or whatever you want to call it library.

Reply:
That was duplicated to avoid cyclic import statements. I've now put the method under `utils.py` :)